src/components/AppProvider/README.md

```diff
@@ -430,13 +430,48 @@ function AppProviderWithAllThemeKeysExample() {
 
 Translations are provided in the locales folder. When using Polaris, you are able to import translations from all languages supported by the core Shopify product and consume them through the `i18n` prop.
 
+At its simplest you can pass the json content from a locale file into `AppProvider`. This approach is suitable when a project has only one specific locale.
```

Comment:
Can we simplify to something like this?

```suggestion
If a project has only one locale, then you can pass the JSON content from the locale file into `AppProvider`. 
```

Reply:
Much nicer! And thank you for getting rid of my use of "simplest". Describing something as simple is a bold assumption and I should know better :D